.github/workflows/ci.yaml

```diff
@@ -281,6 +281,9 @@ jobs:
         if: steps.node_modules-cache-restore.outputs.cache-hit != 'true'
         run: npm ci
 
+      - name: Seed database
```

Comment:
This won't seed the database.  It generates the Prisma client and types.  If this was copied from elsewhere in this file, it should get updated there too for clarity.